.docker/Dockerfile

```diff
@@ -58,18 +58,25 @@ RUN mkdir -p src && \
     . /opt/ros/${ROS_DISTRO}/setup.sh && \
     rosdep install -r --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} -y
 
-RUN . /opt/ros/${ROS_DISTRO}/setup.sh && \
-    apt remove ros-$ROS_DISTRO-moveit*
-
 RUN . /opt/ros/${ROS_DISTRO}/setup.sh && \
     colcon build --mixin release --parallel-workers 1
 
 # Install any other dependencies of this package
-COPY . ${WORKSPACE}/src/moveit_drake
+# NOTE: Copying only the necessary files to not bust cache as much.
+COPY package.xml ${WORKSPACE}/src/moveit_drake/
+COPY moveit_drake.repos ${WORKSPACE}/src/moveit_drake/
+WORKDIR ${WORKSPACE}/src/
+RUN vcs import --force < moveit_drake/moveit_drake.repos
```

Comment:
@kamiradi I had to fix this because the paths were wrong, and the `--force` flag was then needed because this is trying to clone `moveit2`, `moveit_resources`, and `moveit_msgs` as duplicate repos.

I can't say I follow this very well, but just letting you know.